### PR TITLE
Add PDXKBC badge keyboard

### PR DESCRIPTION
**Pull request by fharding1**
## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [ ] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/pdxkbc/config.h [hunk 1]
```diff
@@ -0,0 +1,251 @@
...
+// /*== customize breathing effect ==*/
+//   /*==== (DEFAULT) use fixed table instead of exp() and sin() ====*/
+//   #define RGBLIGHT_BREATHE_TABLE_SIZE 256      // 256(default) or 128 or 64
+//   /*==== use exp() and sin() ====*/
+//   #define RGBLIGHT_EFFECT_BREATHE_CENTER 1.85  // 1 to 2.7
+//   #define RGBLIGHT_EFFECT_BREATHE_MAX    255   // 0 to 255
+// #endif
+
+/* Debounce reduces chatter (unintended double-presses) - set 0 if debouncing is not needed */
+#define DEBOUNCING_DELAY 5
```
Comment (noroadsleft) [inline keyboards/pdxkbc/config.h]: This setting was recently renamed.

```suggestion
#define DEBOUNCE 5
```

File: keyboards/pdxkbc/readme.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
+# pdxkbc
+
+![pdxkbc](https://i.imgur.com/GgNvZcW.jpg)
+
+A macropad made for the Portland Keyboard Club and DEF CON
+
+Keyboard Maintainer: [Franklin Harding](https://github.com/yourusername)  
+Hardware Supported: https://github.com/fharding1/pdxkbc-badge
```
Comment (noroadsleft) [inline keyboards/pdxkbc/readme.md]: This line needs to end in two spaces for GitHub to render this as intended.

```suggestion
Hardware Supported: https://github.com/fharding1/pdxkbc-badge  
```

File: keyboards/pdxkbc/rules.mk [hunk 3]
```diff
@@ -0,0 +1,81 @@
+# MCU name
+#MCU = at90usb1286
```
Comment (noroadsleft) [inline keyboards/pdxkbc/rules.mk]:
```suggestion
```

File: keyboards/pdxkbc/readme.md [hunk 4]
```diff
@@ -0,0 +1,17 @@
+# pdxkbc
+
+![pdxkbc](https://i.imgur.com/GgNvZcW.jpg)
+
+A macropad made for the Portland Keyboard Club and DEF CON
+
+Keyboard Maintainer: [Franklin Harding](https://github.com/yourusername)  
+
```
Comment (noroadsleft) [inline keyboards/pdxkbc/readme.md]: By "two spaces" I meant the kind you get by hitting the spacebar twice. :smile: 

```suggestion
```

File: keyboards/pdxkbc/readme.md [hunk 5]
```diff
@@ -0,0 +1,17 @@
...
+Hardware Supported: https://github.com/fharding1/pdxkbc-badge
```
Comment (noroadsleft) [inline keyboards/pdxkbc/readme.md]:
```suggestion
Hardware Supported: https://github.com/fharding1/pdxkbc-badge  
```

File: keyboards/pdxkbc/readme.md [hunk 6]
```diff
@@ -0,0 +1,17 @@
...
+
```
Comment (noroadsleft) [inline keyboards/pdxkbc/readme.md]:
```suggestion
```

File: keyboards/pdxkbc/readme.md (hunk 4 above)
Comment (fharding1) [reply keyboards/pdxkbc/readme.md]: Right, but github renders this correctly now, doesn't it?

Comment (fharding1) [reply keyboards/pdxkbc/readme.md]: I prefer this style of spacing since the spacing is much more explicit looking at the raw markdown.

Comment (noroadsleft) [reply keyboards/pdxkbc/readme.md]: We can leave it like this.

Comment (fharding1) [reply keyboards/pdxkbc/readme.md]: Thanks!

File: keyboards/pdxkbc/readme.md [hunk 7]
```diff
@@ -0,0 +1,17 @@
+# pdxkbc
+
+![pdxkbc](https://i.imgur.com/GgNvZcW.jpg)
+
+A macropad made for the Portland Keyboard Club and DEF CON
+
+Keyboard Maintainer: [Franklin Harding](https://github.com/yourusername)
```
Comment (zvecr) [inline keyboards/pdxkbc/readme.md]: You might want to set `yourusername` to **fharding1**.

Comment (fharding1) [reply keyboards/pdxkbc/readme.md]: Heh, thanks.

**Comment by fharding1** [comment]
Alright, think I did that right. Never used Configurator before.

**Comment by fharding1** [comment]
Thanks @zvecr and @noroadsleft for the reviews :fire:

File: keyboards/pdxkbc/keymaps/default/keymap.c [hunk 8]
```diff
@@ -0,0 +1,65 @@
...
+      }
+      break;
+    case BADGELIFE:
+      if (record->event.pressed) {
+        SEND_STRING("#badgelife" SS_TAP(X_ENTER));
+      }
+      break;
+    case HACKTHEPLANET:
+      if (record->event.pressed) {
+        SEND_STRING("HACK THE PLANET!" SS_TAP(X_ENTER));
```
Comment (drashna) [inline keyboards/pdxkbc/keymaps/default/keymap.c]: ❤️

**Pull request merged**